conjure/enum.go

```diff
@@ -131,3 +135,27 @@ func enumUnmarshalTextAST(e spec.EnumDefinition, info types.PkgInfo) astgen.ASTD
 	}
 	return newUnmarshalTextMethod(enumReceiverName, transforms.Export(e.TypeName.Name), switchStmt, statement.NewReturn(expression.Nil))
 }
+
+// Goastwriter does not allow values in var declarations, so implement it here.
+// This should be contributed back to goastwriter.
```

Comment:
https://github.com/palantir/goastwriter/pull/58

Reply:
I went ahead and approved the goastwriter change above, so feel free to rebase and I can review this one.

Reply:
Done!